mlflow/utils/file_utils.py

```diff
@@ -228,3 +229,37 @@ def write_to(filename, data):
 def append_to(filename, data):
     with open(filename, "a") as handle:
         handle.write(data)
+
+
+def _copy_project(src_path, dst_path=""):
```

Comment:
You should document that this looks at a docker ignore file if one exists. Should it also do gitignore?